src/OperationalUI/OperationalUI.tsx

```diff
@@ -239,6 +232,35 @@ class OperationalUI extends React.Component<OperationalUIProps, State> {
       </ThemeProvider>
     )
   }
+
+  private pushMessage = (message: IMessage) => {
+    this.setState(prevState => {
+      const previousMessageWithSamePayload = prevState.messages.find(
```

Comment:
We're not really using this value but instead using it as a check. I would propose the following adjustment in order to communicate intent a little clearer:

```suggestion
      const isPreviousMessageWithSamePayload = Boolean(prevState.messages.find(
```

**(Be sure to add the closing `)` on line `240` if you accept this)**